lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -135,6 +135,13 @@ def self.deliver(*args)
     end
   end
 
+  def self.return_result(workspace, options)
+    case options["result_format"]
+    when 'ignore' then options["result_on_success"] || 'Ok'
+    when nil then workspace
```

Comment:
Should `workspace` just be the `else` case?

Are there current plans to use `options["result_on_success"]`?

Reply:
@gmcculloug Since the callback is not looking at the result I am not sure how it figures out if there was an error. We thought this attribute allows the caller to ignore the result but return a precanned value on success which can be checked in the callback.

Reply:
Thanks @mkanoor I'm not against it, just wasn't sure if you had plans to use it with the custom button work.

For my `else` comment above I realized keeping it in the current form makes it obvious that if they user passes a `result_format` they need to specific a supported type or it will return `nil`.  I think that is reasonable.  👍